eng/formatting/format.sh

```diff
@@ -16,7 +16,7 @@ if [ -n "$NATIVE_FILES" ]; then
 fi
 if [ -n "$MANAGED_FILES" ]; then
     # Format all selected files
-    echo "$MANAGED_FILES" | cat | xargs | sed -e 's/ /,/g' | dotnet format --include
+    echo "$MANAGED_FILES" | cat | xargs | sed -e 's/ /,/g' | xargs dotnet format --include
```

Comment:
We added support for `-` (read from stdin), does it not work [either](https://github.com/dotnet/format/issues/1385) with the SDK variant of dotnet-format?
 ```suggestion
    echo "$MANAGED_FILES" | cat | xargs | sed -e 's/ /,/g' | dotnet format --include -
```

Reply:
I did not test that. That would be worth looking into.

Reply:
```suggestion
    echo "$MANAGED_FILES" | cat | xargs | sed -e 's/ /,/g' | dotnet format --no-restore --include -
```

Reply:
With the non-bundled dotnet-format (`dotnet tool install -g format`), all these example work: https://github.com/dotnet/format#examples (especially the pipe ones where we can do stuff like `git ls-files *.cs | dotnet format --include - --folder` or `dotnet show --name-only --pretty="" | dotnet format - ...` etc.

Unfortunately that support was dropped in SDK flavor. Now the only option is "workspace / solution context", which takes ages to format like 2 files in huge repo like runtime..

Reply:
I guess we'll probably have to revisit how this script works then.

At least with this change, we don't dump the `dotnet-format` help page.